Cesium3DTiles/include/Cesium3DTiles/Asset.h

```diff
@@ -0,0 +1,33 @@
+// This file was generated by generate-gltf-classes.
+// DO NOT EDIT THIS FILE!
+#pragma once
+
+#include "Library.h"
+
+#include <CesiumUtility/ExtensibleObject.h>
+
+#include <optional>
+#include <string>
+
+using namespace CesiumUtility;
```

Comment:
It's essential to never include a `using namespace CesiumUtility` in a header file.
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-using-directive